tests/test_output_functions.py

```diff
@@ -0,0 +1,60 @@
+"""Test suite for output_functions.py."""
+
+from gatorgrade.output import output_functions
+
+
+def test_run_commands_and_return_results_returns_correct_results():
+    """Make sure the receive function in output_functions.py runs
+    and returns the correct results in the form of a tuple list"""
+
+    commands = {
+        "shell": [{"description": "Run program", "command": "mdl"}],
+        "gatorgrader": [
+            [
+                "--description",
+                "Check TODOs",
+                "MatchFileFragment",
+                "--fragment",
+                "TODO",
+                "--count",
+                "0",
+                "--exact",
+                "--directory",
+                "../output",
+                "--file",
+                "output_functions.py",
+            ]
+        ],
+    }
+
+    expected_result = [("Check TODOs", True, "")]
+    actual_result = output_functions.run_commands_and_return_results(commands)
+
+    assert expected_result == actual_result
```

Comment:
It would be very useful to have comments that describe the behaviors being tested using the "Given, When, Then" structure. Right now, it's not clear why we expect the actual result to look like the given expected result--a comment could be used to clarify this! 🙂